platform/android/src/native_map_view.cpp

```diff
@@ -171,110 +171,183 @@ void NativeMapView::deactivate() {
  */
 void NativeMapView::invalidate() {
     android::UniqueEnv _env = android::AttachEnv();
-    static auto onInvalidate = javaClass.GetMethod<void ()>(*_env, "onInvalidate");
+    static auto onInvalidate = javaClass.GetMethod<void()>(*_env, "onInvalidate");
     javaPeer->Call(*_env, onInvalidate);
 }
 
 /**
- * From mbgl::RendererBackend. Callback to java NativeMapView#onMapChanged(int).
+ * From mbgl::RendererBackend. Callback to java NativeMapView#onCameraWillChange().
```

Comment:
Actually. This is from `MapObserver` and will only be coming from the main (map) thread. Could you update the comment in the header file as well?